setup.py

```diff
@@ -1,5 +1,9 @@
 from setuptools import setup, find_packages
 from setuptools.command.test import test
+from distutils.util import convert_path
+
+meta = {}
+exec(open(convert_path('vsutil/_metadata.py')).read(), meta)
```

Comment:
I suppose the reason you're not importing it normally is so it doesn't require vapoursynth? If so, I believe that deserves a comment.

Reply:
https://github.com/Irrational-Encoding-Wizardry/vsutil/pull/51/commits/5d7689d039c12e4341325d3d622056e487886737#diff-2eeaed663bd0d25b7e608891384b7298R5-R14

If the module was a single file, then yes, RTD not being able to build vapoursynth would be the reason to not import normally. But, it gets a bit more complicated when it's a package with multiple relative imports in it, that at the start of this script is uninitialized or "partially initialized" which causes circular import issues and the issue we have with `types.py`'s namespace.
I also believe this breaks installing it with pep as it will attempt to run the uninitialized module before knowing what dependencies are needed (in this case just vapoursynth).

Reply:
Yes, it would fail for setup.py and for RTD, neither of which would have `vapoursynth` importable. The comment might be a bit verbose, but that's better than no comment imo.

Reply:
The error message from the python console in pycharm mentions circular imports because of our types submodule trying to import from enum which imports from the stdlib types. Since the package isn't built, stdlib enum tries to import from our types instead as it doesn't know it's namespace is actually vsutil.types.